source_account/terminate_ec2.tpl

```diff
@@ -0,0 +1,3 @@
+echo ** terminating EC2 ${instance_id} in region ${region} **
```

Comment:
you could add name this file `terminate_ec2.tpl.sh` to get syntax colouring and other CI checks.